src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2793,14 +2816,6 @@ namespace NServiceBus.Unicast
     [System.ObsoleteAttribute("Not a public API. Please use `MessageHandlerRegistry` instead. Will be removed in" +
         " version 7.0.0.", true)]
     public interface IMessageHandlerRegistry { }
-    public class MessageContext : NServiceBus.IMessageContext
```

Comment:
>I think users can't really access that class so it should be safe to just make it private?


I'd say to obsolete it anyway, doesn't cost us much and it will be semver compliant